tests/performance/specs/front-end-classic-theme.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { basename, join } from 'path';
+import { writeFileSync } from 'fs';
+
+/**
+ * WordPress dependencies
+ */
+import { activateTheme, createURL } from '@wordpress/e2e-test-utils';
+
+describe( 'Server Timing – Twenty Twenty One', () => {
+	const results = {
+		wpBeforeTemplate: [],
+		wpTemplate: [],
+		wpTotal: [],
+	};
+
+	beforeAll( async () => {
+		await activateTheme( 'twentytwentyone' );
```

Comment:
Have we validated that this _does not_ set up the theme's starter content? TT1 is one of the themes with starter content, and we have to avoid that it is being added for our test site. Otherwise we get inconsistent test data between the 2 themes.

Most importantly the TT1 starter content would change the home page to be a static front page (`front-page` template) instead of the list of posts (`home` template) (see https://wp-themes.com/twentytwentyone/ which has that starter content), so we need to avoid that here.

Reply:
@felixarntz I test all the steps in local system and it will now add theme's starter content for TT1.

cc. @joemcgill

Reply:
@mukeshpanchal27 Sorry, do you mean "now" or "not"? If the theme starter content is now added, we will need to change that. The theme starter content should _not_ be added, since otherwise it makes the test content inconsistent.

Reply:
I just ran through this locally, and it does not seem to trigger starter content, so I think we're good.